docs/release/release_0_4_16.md

```diff
@@ -26,15 +26,38 @@ After discussion in [#4102](https://github.com/napari/napari/pull/4102) and [#41
 
 
 - Calling `viewer.open` *without* passing a plugin will result in an error if you have not saved a reader preference for that file pattern *and* multiple plugins can claim the file
-    - You can address this error by associating a preference for the file pattern, or calling `viewer.open(file_path, plugin=...)
-- A preferred reader failing to read your file will result in an error
+    - To save a preference for a file pattern in Python, use:
+
+    ```python
+    from napari.settings import get_settings
+    get_settings().plugins.extension2reader['*.tif'] = napari_tifffile
```

Comment:
from the example, it's unclear what the values are here.   is `napari_tifffile` a module?  a string?

Reply:
thanks for catching that, they're strings. Have updated now.